src/SFML/Window/Win32/InputImpl.cpp

```diff
@@ -595,10 +627,22 @@ Keyboard::Scancode delocalize(Keyboard::Key key)
 ////////////////////////////////////////////////////////////
 String getDescription(Keyboard::Scancode code)
 {
-    const WORD winCode = sfScanToWinScanExtended(code);
-    const int  bufSize = 1024;
-    WCHAR      name[bufSize];
-    const int  result = GetKeyNameText(winCode << 16, name, bufSize);
+    // Try to translate the scan code to a consumer key
+    if (const auto consumerKeyName = sfScanToConsumerKeyName(code))
+        return consumerKeyName->data();
```

Comment:
I now see that the return value is only used here to create an `sf::String`. Would it be easier if `sfScanToConsumerKeyName` already returned an `sf::String`?